pkg/handler/api.go

```diff
@@ -169,3 +183,21 @@ func ParsePagination(c echo.Context) api.PaginationData {
 	}
 	return pageData
 }
+
+func ParseFilters(c echo.Context) api.FilterData {
+	filterData := api.FilterData{
+		Search:  DefaultSearch,
+		Arch:    DefaultArch,
+		Version: DefaultVersion}
+	err := echo.QueryParamsBinder(c).
+		String("search", &filterData.Search).
+		String("arch", &filterData.Arch).
+		String("version", &filterData.Version).
+		BindError()
+
+	if err != nil {
+		log.Fatal().Err(err)
```

Comment:
log.Fatal() actually causes the program to exit after the message is logged which is bad! I see that ParsePagination is doing that too, mind updating both to log.Error()?

Reply:
Roger. Still dealing with seed changes on tests.